### DISCUSSION TITLE
Load (PyTorch|Tensorflow) model, add or drop layers, and finetune

### DISCUSSION THREAD

**Discussion by kgoderis** [deepjavalibrary/djl#3022] [Q&A]
I am looking for an example that
1. Loads an existing model, for example a PyTorch from HuggingFace
2. Drops some layers from that model
3. Add some new layers to that model
4. Trains that model

Can an existing model be embedded in a new model?

**Comment by zachgk** [comment]
Take a look at our [transfer learning examples](https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/training/transferlearning) such as the [ResNet Cifar10 example](https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/training/transferlearning/TrainResnetWithCifar10.java). However, support for removing the last block is only available in MXNet right now
